### PR TITLE
[MXNET-793] Virtualized ARMv7 with Qemu CI integration

### PR DESCRIPTION
**Pull request by larroy**
## Description ##

This runs a minimal validation of MXNet on ARMv7 

Broader validation is pending on test refactoring which is out of the scope of virtual testing.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by anirudhacharya** [comment]
@mxnet-label-bot [pr-awaiting-review]

File: Jenkinsfile [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 // mxnet libraries
 mx_lib = 'lib/libmxnet.so, lib/libmxnet.a, 3rdparty/dmlc-core/libdmlc.a, 3rdparty/tvm/nnvm/lib/libnnvm.a'
+
+// Python wheels
+mx_pip = 'mxnet_dist/*.whl'
```
Comment (marcoabreu) [inline Jenkinsfile]: Are you sure about that path? in my experiments, it was in a different path: https://github.com/marcoabreu/incubator-mxnet/blob/add-raspberry-ci/tests/nightly/JenkinsfileEdge#L21

File: Jenkinsfile [hunk 2]
```diff
@@ -941,6 +946,16 @@ core_logic: {
           }
         }
       }
+    },
+    'ARMv7 QEMU': {
+      node(NODE_LINUX_CPU) {
+        ws('workspace/ut-armv7-qemu') {
+          timeout(time: max_time, unit: 'MINUTES') {
+            utils.unpack_and_init('gpu', mx_pip)
```
Comment (marcoabreu) [inline Jenkinsfile]: did you mean armv7 instead of gpu? Might be a copy&paste error

File: Jenkinsfile [hunk 3]
```diff
@@ -941,6 +946,16 @@ core_logic: {
...
+            sh "ci/build.py --docker-registry ${env.DOCKER_CACHE_REGISTRY} -p test.arm_qemu ./runtime_functions.py run_ut_py3_qemu"
```
Comment (marcoabreu) [inline Jenkinsfile]: Is it possible to use our docker_run utility instead?

Comment (larroy) [reply Jenkinsfile]: if you look closely, not ATM. because of the runtime_functions.sh which is hardcoded.

Comment (marcoabreu) [reply Jenkinsfile]: Ah, gotcha. We can follow up on that in a separate PR. Can you create an issue so we don't forget it?

**Pull request merged**